decidim-core/lib/decidim/resourceable.rb

```diff
@@ -108,6 +107,16 @@ def resource_visible?
           true
         end
       end
+
+      # Public: Whether the permissions for this object actions can be set at resource level.
+      def allow_resource_permissions?
+        false
```

Comment:
This is the only line without coverage, because this method is overriden by all resourceables models.

I can change it to 
```ruby
def allow_resource_permissions?; end
```
and it would work well, but I think is would be less clear. 

I also can add a new dummy resource class that doesn't override this method, but I think that it doesn't worth it.

Reply:
It's OK for me as it is! 😄

Reply:
Does it need to be overriden? If so maybe add a `raise NotImplementedError`?

Reply:
`NotImplementedError` is not intended for this use case, I think... 

https://ruby-doc.org/core-2.2.0/NotImplementedError.html

Reply:
@oriolgual it doesn't need to be overridden. By default, resources shouldn't allow to set permissions at that level.

Also, I just realized that there are a lot of other resourceable models, we can just add a test for any of that model and complete the coverage.